### ISSUE TITLE
Imagenet1k Full Spectrum Accuracy is lower than the Leadboard

### ISSUE THREAD

**Issue by Rlyslata** [Jingkang50/OpenOOD#310]
Dear Author, 
      DESC: The running script is completely the original version of openood1.5 - no modification (msp && resnet50). From the attachment "openood run log.txt", the accuracy rate in the ood task is 76%, which is correct, indicating that the weights have been loaded. However, in the "Full Spectrum" task, the result shows an accuracy of 46%, which is lower than the 54% of MSP-CrossEntropy in "Leadboard".
      BenchMarkList is downloaded from the OpenOOD1.5 github homepage link, and the Imagenet verification set is downloaded from Academic Torrents. I am not sure whether it is a problem with the imagenet verification set or the CSID data set.
      If it is a problem with the imagenet validation set, is the solution to download the complete official imagenet data set (train, test, val)?
[openood msp log.txt](https://github.com/user-attachments/files/26740606/openood.msp.log.txt)

**Comment by Rlyslata** [comment]
I have found it, LeadBoard model didn't use imagenet-es as csid data.
